tools/depends/target/libdovi/LIBDOVI-VERSION

```diff
@@ -0,0 +1,7 @@
+LIBNAME=libdovi
+BASE_URL=https://github.com/quietvoid/dovi_tool/archive
+VERSION=3.1.2
```

Comment:
@quietvoid im just putting sources on our mirrors. Ive added rustup and cargo-c, so if you can remove the BASE_URL from their version files.

However i just wanted to confirm you want this version still. I saw you released a 3.2.0 version last month. Let me know what version you want to roll with and i'll get it on mirrors so we can merge this.

Reply:
The only difference would be that some warnings are fixed, but no meaningful changes.

We could just bump everything as well, `cargo-c` and `libdovi` at least had new tags.
The Rust latest stable also got a new version, but it doesn't rely on mirrors.

Reply:
Ok, we can just go with 3.1.2, and always update later if you desire.

Can you remove the BASE_URL from this, CARGO-C-VERSION and RUSTUP-VERSION respectively and then we'll be good to merge

Reply:
Does `ARCHIVE` need to be renamed too? On the mirrors I presume they have the name now?
There were concerns before because the name was just the version (`BASE_URL` limitation), and the suggestion was to always do
```
SOURCE=$(APPNAME)-$(VERSION)
ARCHIVE=$(SOURCE).tar.gz
```